datenguide_python/query_builder.py

```diff
@@ -229,7 +367,7 @@ def __init__(
     @classmethod
     def regionQuery(
```

Comment:
Although not part of this pull request I was wondering whether we need the **Query** in region**Query** as it leads to calls like `Query.regionQuery(...)` which appears redundant. (same goes for all regions). If we decide to keep it pep8 would suggest `region_query` i think.

Reply:
agreed. I wasnt happy with the naming either.